### ISSUE TITLE
Tensorboard for this repo

### ISSUE THREAD

**Issue by rodrigo2019** [yhenon/keras-frcnn#165]
It is working for me

_import keras_frcnn.roi_helpers as roi_helpers
from keras.utils import generic_utils_
**from keras.callbacks import TensorBoard**
. . .

_model_all = Model([img_input, roi_input], rpn[:2] + classifier)_
**tbCallBack = TensorBoard(log_dir='log', histogram_freq=1,  
          write_graph=True, write_images=True)
tbCallBack.set_model(model_all)**
. . .

_class_mapping_inv = {v: k for k, v in list(class_mapping.items())}
print('Starting training')_

**def write_log(callback, names, logs, batch_no):
        for name, value in zip(names, logs):
            summary = tf.Summary()
            summary_value = summary.value.add()
            summary_value.simple_value = value
            summary_value.tag = name
            callback.writer.add_summary(summary, batch_no)
            callback.writer.flush()**
        
**train_names = ['train_loss_rpn_cls', 'train_loss_rpn_reg','train_loss_class_cls','train_loss_class_reg','train_total_loss','train_acc']
val_names = ['val_loss_rpn_cls', 'val_loss_rpn_reg','val_loss_class_cls','val_loss_class_reg','val_total_loss','val_acc']**

. . .
                _iter_num = 0
                start_time = time.time()_
                **write_log(tbCallBack, train_names, [loss_rpn_cls,loss_rpn_regr,loss_class_cls,loss_class_regr,curr_loss,class_acc], epoch_num)**

**Comment by Abhijit-2592** [comment]
@rodrigo2019  could you please provide it in code format?

**Comment by rodrigo2019** [comment]
[train_frcnn.zip](https://github.com/yhenon/keras-frcnn/files/1306972/train_frcnn.zip)

**Comment by Abhijit-2592** [comment]
thanks :)

**Comment by mrqorib** [comment]
@rodrigo2019 is the darknet network you put in the train file is the network YOLO used?

**Comment by rodrigo2019** [comment]
@mrqorib, the base network are very similar.
I changed a little bit the darknet, but I think it is almost the same network yet

[darknet.zip](https://github.com/yhenon/keras-frcnn/files/1454521/darknet.zip)

**Comment by mrqorib** [comment]
@rodrigo2019 Thank you Rodrigo! Btw, I want to ask, does your code save 2 type of model every 3 batch (one with better training loss and one with better validation loss)? Is that the only difference or is there any other difference with the original in this repo?
